### DISCUSSION TITLE
Get your Galaxy Badge

### DISCUSSION THREAD

**Discussion by MatthysDev** [community/community#101991] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The fastest way to get your galaxy badge here

How should i do to have the galaxy badge ? Reply to get it 🤔

**Comment by samus-aran** [comment]
Hi @MatthysDev 

We highly discourage badge hunting on this discussion board as it violates GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibiting coordinated or inauthentic activity like rapid questions and answers.

As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
